src/modules/sensors/voted_sensors_update.cpp

```diff
@@ -152,9 +152,9 @@ void VotedSensorsUpdate::parameters_update()
 	_temperature_compensation.parameters_update(_hil_enabled);
 
 	/* gyro */
-	for (int topic_instance = 0; topic_instance < _gyro.subscription_count; ++topic_instance) {
+	for (uint8_t topic_instance = 0; topic_instance < _gyro.subscription_count; ++topic_instance) {
```

Comment:
I don't particularly like this but I suppose it's to save RAM in the subscriptions, right? Are you sure it doesn't get padded away anyway?

Reply:
It's not really an optimization, it's just to keep the types consistent.